readme.md

```diff
@@ -39,23 +39,28 @@ $ npm -v
 * **Question:** What is the difference between forking and cloning a repository as opposed to just cloning a repository?
 
 * **Your Answer:**
+  `git fork` - creates a safe environment for you to write code and create experiments without impacting the master branch. You could merge this code in later.
```

Comment:
This is all true, but you could do this by just creating a branch as well. Forking makes a copy of the repository from one account to another account on GitHub.